requirements_tools/upgrade_requirements.py

```diff
@@ -93,19 +98,19 @@ def cleanup_dir(dirname):
 
 def make_virtualenv(args):
     with cleanup_dir(tempfile.mkdtemp()) as tempdir:
-        venv, python, pip = dirs(tempdir)
+        venv, python, pip, pip_tool = dirs(tempdir, args.pip_tool)
         print_call(
             sys.executable, '-m', 'virtualenv', venv,
             '-p', args.python, '--never-download',
         )
 
-        def pip_install(*argv):
+        def pip_install(pip, *argv):
             print_call(pip, 'install', '-i', args.index_url, *argv)
 
         # Latest pip installs python3.5 wheels
-        pip_install('pip', 'setuptools', '--upgrade')
-        pip_install('-r', 'requirements-minimal.txt')
-        pip_install('-r', 'requirements-dev-minimal.txt')
+        pip_install(pip, 'pip', 'setuptools', args.pip_tool, '--upgrade')
```

Comment:
this assumes that the package and executable are the same, which isn't necessarily true

Taking precedence from `check-all-wheels`, please match the arguments and approach there: https://github.com/Yelp/requirements-tools/blob/1a6e756c98a73f9fa9829721be2ce402d917cf10/requirements_tools/check_all_wheels.py#L21-L22